src/pdn/doc/define_pdn_grid.md

```diff
@@ -0,0 +1,104 @@
+## define_pdn_grid
+
+### Synopsis
+For specifying a power grid over the stdcell area
+```
+  % define_pdn_grid
+    -type stdcell \
+    [-name name] \
+    [-rails list_of_rail_specifications] \
+    [-straps list_of_strap_specifications] \
+    [-pins list_of_pin_layers] \
+    [-connect list_of_connected_layer_pairs] \
+    [-starts_with (POWER|GROUND)]    
+```
+For specifying a power grid over macros in the design
+```
+  % define_pdn_grid
+    -type macro \
```

Comment:
how do you define this for a specific macro type? ie. does this need a `-master`?

Reply:
The ways that this is intended to work (in order of priority)
- Use instance attribute to associate a given grid with specific instances
- Use macro attribute to associate a given grid with specific macro cells
- Use orient attribute to associate a given grid for instances with specific orientations
- Otherwise use a grid with none of the above specified.
So, need to add -inst and -master options for the first two of these to go with -orient alreadry defined.